CondFormats/HGCalObjects/interface/HGCalMappingCellIndexer.h

```diff
@@ -149,7 +149,7 @@ class HGCalMappingCellIndexer {
   size_t getNErxExpectedFor(size_t typecodeidx) const { return maxErx_[typecodeidx]; }
 
   constexpr static char maxHalfPerROC_ = 2;
-  constexpr static uint16_t maxChPerErx_ = 39;  //36 channels + 1 calib + 2 CM
+  constexpr static uint16_t maxChPerErx_ = 37;  //36 channels + 1 calib + 2 CM
```

Comment:
@pfs Do we want to make this change?
Currently changing it to 37 leads to a crash in `testMappingModuleIndexer_cfg.py`

Reply:
Then we need re-basing to get this change in